gradle.properties

```diff
@@ -0,0 +1,10 @@
+# Project information
+group                      = "com.ekino.oss.gradle.plugin"
```

Comment:
Not sure you need double quotes for value here

Reply:
it seems they are added to the builded jar filename

Reply:
You're right! It's fixed ^^'